app/uk/gov/hmrc/tariffclassificationfrontend/models/AssigneeCount.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.models
+
+case class AssigneeCount(op: Operator, count: Int) {
+  def name: String = op.safeName
+  def firstName: String = name.split(" ").head
```

Comment:
You are calling `name.split(" ")` twice.
You could save `name.split(" ")` in a lazy private val.

Reply:
What's the need to split up the firstname and surname?
I don't think simply splitting on spaces will work all the time

Reply:
It's used for sorting by last name then first name.   There are tests that check that users with a single name will have the same first and last name.  And if there is no name for some reason then it should resolve to "PID xx"